.gitignore

```diff
@@ -4,3 +4,5 @@ build/
 /metrictank
 /cmd/mt-split-metrics-by-ttl/mt-split-metrics-by-ttl
 /cmd/mt-store-cat/mt-store-cat
+/cmd/whisperImporter/whisperReader/main
+/cmd/whisperImporter/chunkWriter/main
```

Comment:
let's stick to the convention of how to name binaries, and to not use subdirectories like this. suggestion:
```
/cmd/mt-whisper-to-chunks
/cmd/mt-chunks-to-cassandra
```